packages/amplify-category-geo/resources/custom-device-location-tracking-resource-handler.js

```diff
@@ -0,0 +1,33 @@
+const response = require('cfn-response');
```

Comment:
Does this resource file need an associated `package.json` file?

Reply:
Hm I'm not too familiar with why it would need a `package.json`? Can you elaborate? We copied the same file structure for these resource files as the other geo resources

Reply:
It looks like it will be fine without the `package.json` because it's being used as a custom resource: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-lambda-function-code-cfnresponsemodule.html